ui/package.json

```diff
@@ -59,7 +59,7 @@
     "@bower_components/jasmine": "pivotal/jasmine#>=2",
     "@bower_components/jasmine-jquery": "velesin/jasmine-jquery#2.1.0",
     "@bower_components/q": "kriskowal/q#1.0.1",
-    "bahmni-form-controls": "^0.93.18",
+    "bahmni-form-controls": "^0.93.20",
```

Comment:
@rahu1ramesh Please commit the changes on yarn.lock file as well for the version upgrade. CI pipelines use a frozen lock file and without updating yarn.lock, the dependency won't get updated.